.github/workflows/release.yml

```diff
@@ -71,36 +71,41 @@ jobs:
           - os: macos-13
             target: x86_64-apple-darwin
             suffix: ""
+            cross: false
           - os: macos-14
             target: aarch64-apple-darwin
             suffix: ""
+            cross: false
           - os: ubuntu-latest
             target: x86_64-unknown-linux-gnu
             suffix: ""
+            cross: false
           # FIXME: deno doesn't support musl today https://github.com/denoland/deno/issues/3711
           # - os: ubuntu-latest
           #   target: x86_64-unknown-linux-musl
           #   suffix: ""
+          #   cross: false
           - os: ubuntu-latest
             target: aarch64-unknown-linux-gnu
             suffix: ""
+            cross: true
     steps:
```

Comment:
Just curious, what's the main advantage of `cross`?

Reply:
It allows us to build arm64 binaries on x86_64.